src/js/App.js

```diff
@@ -60,5 +66,20 @@ window.App = Backbone.View.extend({
             _.delay(this.emptyDetailsView, 200);
             _.delay(this.fadeInListView, 200);
         }
+    },
+
+    slideOutStatsRegion: function() {
+        var $statsRegion = $('.stats');
+        var $mapRegion = $('#map');
```

Comment:
Is it possible to search the `$el` of `mapView` to find these elements? That should be faster than a fresh search of the DOM.

Reply:
Tried this out and we can do it with the MapView, but the stats region isn't a Backbone view. I made the `$statsRegion` assignment in the App's initializer, though, so $ dom stuff will happen prior to this call.